### PR TITLE
Added support for more RabbitMQ servers

### PR DESCRIPTION
**Pull request by rpliva**
This solves issue #13

### CODE REVIEW

File: src/main/scala/cz/alenkacz/marathon/scaler/Application.scala [hunk 1]
```diff
@@ -11,13 +11,14 @@ trait Application {
   def queueName: String
   def maxMessagesCount: Int
   def maxInstancesCount: Option[Int]
+  def serverName: String
```
Comment (alenkacz) [inline src/main/scala/cz/alenkacz/marathon/scaler/Application.scala]: I find server name too generic, maybe rmqServerName?

**Comment by alenkacz** [comment]
since most people probably do not use multiple rmq servers I would prefer if the default choice was:
- name not being necessary
- when name is not provided in label, it will still work (if there are not multiple RMQ instances configured)

If you agree, please make the changes. if you do not want to, I can do it myself.

Btw. Thanks for the pull request! :)

**Review by alenkacz — COMMENTED** [review]
since most people probably do not use multiple rmq servers I would prefer if the default choice was:
- name not being necessary
- when name is not provided in label, it will still work (if there are not multiple RMQ instances configured)
  If you agree, please make the changes. if you do not want to, I can do it myself.

Btw. Thanks for the pull request! :)

**Comment by rpliva** [comment]
The name is optional for the RabbitMQ configuration and also for applications definitions

**Comment by alenkacz** [comment]
Thanks :)

**Pull request merged**